BUILD.bazel

```diff
@@ -32,6 +32,11 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "krnlmon_options",
+    srcs = ["krnlmon_options.h"],
```

Comment:
Where is the `krnlmon_options.h` file? I dont see it committed in this PR

Reply:
Aw, nuts. I forgot to add it.